requirements/pytorch/extra.txt

```diff
@@ -6,6 +6,6 @@ matplotlib>3.1, <3.5.3
 omegaconf>=2.0.5, <2.3.0
 hydra-core>=1.0.5, <1.3.0
 jsonargparse[signatures]>=4.12.0, <4.14.0
-gcsfs>=2021.5.0, <2022.8.0
+gcsfs>=2021.5.0, <2022.9.0
```

Comment:
I just noticed that this requirement is unnecessary. It was added to run gcsfs tests, but the tests don't run at all in our CI: https://github.com/Lightning-AI/lightning/pull/7946#discussion_r984102808

So I suggest removing it and the tests added above.
Alternatively, I suggest moving this requirement into the `test` requirements

Reply:
Please! As little dependencies as possible, so that as little things as possible can break :)